### ISSUE TITLE
[CoE Starter Kit - QUESTION] What is the DLP Evaluation Status in Power Apps Apps in Power Platform Admin View?

### ISSUE THREAD

**Issue by jbst2018** [microsoft/coe-starter-kit#3281]
### What is your question?

hello

What is the DLP Evaluation Status in Power Apps Apps in Power Platform Admin View?
If the status is NonCompliant, does it mean that it does not comply with the DLP policy created by the organization?

Also, some apps haven't updated their DPL Last Evaluation Date since a month ago.
How long is the update interval?

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

June 2022

**Comment by manuelap-msft** [comment]
Hello,

that's correct, if the status is NonCompliant the app is not compliant with DLP policies that are applied to this environment. If you upgrade to the July version, we've added additional information here to tell you which DLP policies are violated, and which connectors in the app are violating it. 

DLP Status is evaluated all the time by the product - e.g. if you change a DLP policy and apps are impacted, they would almost instantly stop working: https://docs.microsoft.com/en-us/power-platform/admin/dlp-impact-policies-apps-flows

However, if no changes are made to the app or DLP policies in the tenant, the last evaluation date may be from a while ago - as if nothing changes, no new evaluation happens. 

Hope that helps,
Manuela

**Comment by jbst2018** [comment]
Hi, Manuela
Thank you for your support.
I understand.

**Comment by jbst2018** [comment]
Hello.

Let me ask another question.
When I upgraded to the July version, I got additional information as you said.
The sample app "Milestone" of Teams Store is displayed.
 It says "shared_commondataservice" in the DLP Violation Details, which seems to be a Microsoft Dataverse (legacy).
![image](https://user-images.githubusercontent.com/109139057/181691441-ea9d975f-5c8c-469e-b77c-9d59e6250518.png)
![image](https://user-images.githubusercontent.com/109139057/181691701-0f0ce83d-ce07-4515-9797-a4af6cd9eda6.png)

But in Connection References, it's a Microsoft Dataverse,not Microsoft Dataverse (legacy).
Which is correct?
![image](https://user-images.githubusercontent.com/109139057/181691929-a6638b03-d9cc-49a4-af0e-c9473054a034.png)

Thanks.
